config/initializers/devise.rb

```diff
@@ -0,0 +1,320 @@
+# frozen_string_literal: true
+
+# Assuming you have not yet modified this file, each configuration option below
+# is set to its default value. Note that some are commented out while others
+# are not: uncommented lines are intended to protect your configuration from
+# breaking changes in upgrades (i.e., in the event that future versions of
+# Devise change the default values for those options).
+#
+# Use this hook to configure devise mailer, warden hooks and so forth.
+# Many of these configuration options can be set straight in your model.
+Devise.setup do |config|
+  # The secret key used by Devise. Devise uses this key to generate
+  # random tokens. Changing this key will render invalid all existing
+  # confirmation, reset password and unlock tokens in the database.
+  # Devise will use the `secret_key_base` as its `secret_key`
+  # by default. You can change it below and use your own secret key.
+  # config.secret_key = 'd0f4b0ffcd9dbdb258d555eb97d578e8f73de9fa7322660dd135d2f88e704fe5e17466659b8c7302ba1be71b56e086c90eb370a324e950a0b3fe11e2f92f9c49'
```

Comment:
We ought to change this. Maybe add a TODO if you don't want to do it now so we don't forget?